src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/markduplicates/MarkDuplicatesSpark.java

```diff
@@ -313,7 +313,7 @@ protected void runTool(final JavaSparkContext ctx) {
         if (headerMap.size() > 1) {
             final Optional<Map.Entry<String, SAMFileHeader>> badlySorted = headerMap.entrySet()
                     .stream()
-                    .filter(h -> treatAsReadGroupOrdered(h.getValue(), treatUnsortedAsOrdered))
+                    .filter(h -> !treatAsReadGroupOrdered(h.getValue(), treatUnsortedAsOrdered))
```

Comment:
woops, sorry

Reply:
whats worse is that I actually caught this yesterday but forgot to push the branch until today... sigh...